src/quo2/components/calendar/calendar_day/style.cljs

```diff
@@ -0,0 +1,73 @@
+(ns quo2.components.calendar.calendar-day.style
+  (:require
+    [quo2.foundations.colors :as colors]
+    [quo2.foundations.typography :as typography]))
+
+(def wrapper
+  {:flex            1
+   :margin-vertical 2
+   :justify-content :center
+   :align-items     :center})
+
+(def container-base
+  {:align-items     :center
+   :justify-content :center
+   :border-radius   10
+   :height          32
+   :width           32})
+
+(defn text-base
+  []
+  (-> typography/paragraph-2
+      (merge typography/font-medium)
+      (merge
+       {:color      (colors/theme-colors colors/neutral-100 colors/white)
+        :text-align :center})))
+
+(defn in-range-background
```

Comment:
hmm, in the figma this variant is not part of the designs fore this component and the range is handled in the calendar component - I understand this approach os easier but this is something we should discuss with designers. either they update the designs or we adjust to match 👍

Reply:
yes we don't have it but as i mentioned in the the group we need it as it's hard to apply style to row, i believe it should be an additional props for item as it has some specific feature like left radius for starting range and left radius for ending range